pyproject.toml

```diff
@@ -32,3 +32,19 @@ include = [
 
 [tool.hatch.build.targets.wheel.force-include]
 "./configs" = "sgm/configs"
+
+[tool.hatch.envs.ci]
+# Skip for now, since requirements.txt is used by scripts and includes the project
+# This should be changed when dependencies are handled by Hatch
+skip-install = true 
+
+dependencies = [
+    "pytest"
+]
+
+[tool.hatch.envs.ci.scripts]
+test-inference = [
+    "pip install torch==2.0.1+cu118 torchvision==0.15.2+cu118 torchaudio==2.0.2+cu118 --index-url https://download.pytorch.org/whl/cu118",
```

Comment:
`torchaudio` isn't required by the repo as far as I'm aware.

Reply:
true this not used

Reply:
It's present in the requirements.txt, which is why it's here - could be removed from both, but dependency cleanup might be best left for another PR.

Reply:
Sure. Dependency cleanup is pending on #44 and #50.